dev/com.ibm.ws.logging/src/com/ibm/ws/logging/data/KeyValueBooleanPair.java

```diff
@@ -73,6 +80,11 @@ public boolean getBooleanValue() {
         return value;
     }
 
+    @Override
+    public ArrayList<KeyValuePair> getList() {
+        return null;
```

Comment:
this should throw an UnsupportedOperationException (see other methods in this class).  Please make this change in all applicable classes.